spec/models/genre_spec.rb

```diff
@@ -4,4 +4,14 @@
   it { should validate_presence_of(:name) }
 
   it { should validate_uniqueness_of(:name) }
+
+  describe "#find" do
+    it "shoudl find genre by slug name" do
+      romance = create(:genre, name: "Romance")
+
+      genre = Genre.find('romance')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.